packages/patternfly-4/react-core/src/components/AlertGroup/examples/alert-group.css

```diff
@@ -0,0 +1,3 @@
+.ws-react-c-alertgroup .pf-c-alert-group__item {
+  margin-bottom: 21px;
```

Comment:
Why do we need this? It isn't an accurate representation of alert group spacing.

Reply:
I saw it in the alert component and thought we'd want to keep the same spacing. Happy to take it out! :)